cogeo_mosaic/backends/http.py

```diff
@@ -45,9 +46,17 @@ def write(self):
         """Write mosaicjson document."""
         raise NotImplementedError
 
-    def update(self):
+    def update(
+        self,
+        features: Sequence[Dict],
+        accessor: Callable,
+        overwrite: bool = False,
+        **kwargs: Any
+    ):
         """Update the mosaicjson document."""
-        raise NotImplementedError
+        self._update(features, accessor, **kwargs)
+        if overwrite:
+            warnings.warn("Overwrite is not possible for http backend")
```

Comment:
so if overwrite in set to `True` we will update the mosaic internally but with the actual CLI, it won't be printed